src/lib/netlist/FAQ.md

```diff
@@ -0,0 +1,150 @@
+usage.
+
+And I am convinced we would see more BTANBs.
+
+### How do frontiers work?
+
+Frontiers divides a netlist into sub netlists, i.e. a number of smaller netlists.
+Frontiers work best if you have a low impedance to high impedance transition. 
+This is best illustrated by an example. Consider the following mixing stage
+	
+	                 R1
+	      S1 >-----1RRRR2---------+
+	                              |
+	                 R2           |
+	      S2 >-----1RRRR2---------+----------> Out
+	                              |
+	                              R
+	                           R3 R
+	                              R
+	                              |
+	                             GND
+	
+With `OPTIMIZE_FRONTIER(R2.2, R3, R2)` this becomes:
+	
+	                 R1
+	      S1 >-----1RRRR2--------------------------------+
+	                                                     |
+	                       ##########################    |
+	                 R2    #                    R2  #    |
+	      S2 >-----1RRRR2-----+-->AnIn AnOut>--RRRR------+----------> Out
+	                       #  |                     #    |
+	                       #  R                     #    R
+	                       #  R R3                  # R3 R
+	                       #  R                     #    R
+	                       #  |                     #    |
+	                       # GND          Frontier  #   GND
+	                       #                        #
+	                       ##########################
+	
+As a result, provided there are no other connections between the parts
+generating S1 and S2 the "S2 part" will now have a separate solver.
+
+The size (aka number of nets) of the solver for S1 will be smaller.
+The size of the solver for S2 and the rest of the circuit will be smaller
+as well.
+
+Frontiers assume that there is feedback from the "out" terminal to the "in" 
+terminal. This is a safe assumption is the "in" terminal e.g. is connected to an 
+op-amp output (typically < 500 Ohm) and R2 in the example above is in the 10 KHz range.
+
+## MAME specific
+
+### Is there an example of a netlist that takes input from an AY-8910/2 and handles it?
+
+There are quite a couple:
+
+- nl_kidniki.cpp
+- nl_konami.cpp
+- 1942.cpp
+
+Basically the AY8910 driver has to be configuredr to push resistor values to the
+input stream. The konami drivers (scramble, frogger and friends) do that and 
+should be used as a reference. 1942 connects outputs and may be an even better example.
+
+## Models
+
+### Is there are JFET model?
+
+No, there is currently no JFET model in netlist. They are close to mosfets so 
+you may try a generic n-mosfet with a negative trigger voltage. Example:
+
+	MOSFET(Q21, "NMOS(VTO=-1.0)")
+
+Writing an analog model is a very time-consuming task. JFETs are typically used 
+in switching applications and consequently the simplified model above should 
+deliver reasonable results.
```

Comment:
I take issue with this statement – JFETS are typically used as high-impedance amplifier inputs.  They make rather poor switches and aren’t frequently used in that application.

Reply:
Removed this.

Reply:
In a bit more detail, a JFET isn’t a very good switch because of the high impedance of the source/drain output.  In its linear region, it exhibits forward transconductance, i.e. a change in gate voltage produces a change in source-drain current.

The gate-channel junction is kept reverse biased, so only a small leakage current flows through the gate.  JFETs have far lower gate capacitance than MOSFETs as they lack of the insulating oxide layer that acts as a dielectric material.  This makes them useful as an amplifier where the signal source has very low current output capability.

Another significant JFET application is a simple constant current source.  A JFET with a resistor connected between the gate and source will act as a constant current source.  If as current increases, the gate-source voltage increases, reducing drain-source conductance.  This provides negative feedback, giving a very high effective output impedance.

Reply:
I copied the above. I hope to reuse it somewhere in the documentation later. Thanks a lot.